CHANGELOG.md

```diff
@@ -8,15 +8,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Changed
-
+- Updated documentation
 - Increase container memory resource requests and limits
 
 ## [0.4.0] - 2022-09-21
 
-### Added
```

Comment:
Did you mean to remove this?

Reply:
:( I did not. Must've happened during conflict resolution.